client/Styles/App.css

```diff
@@ -0,0 +1,29 @@
+.App {
```

Comment:
Watch your indentation of `.css` files. The second line through the end all seem to be indented one notch more than it should be.